.github/workflows/lts-branch-create-pull-request.yaml

```diff
@@ -0,0 +1,71 @@
+name: Create PR for LTS Branch
+on:
+  push:
+    branches:
+    - 'sync-apis/**/**'
+
+jobs:
+  create-pr-for-api-sync-branches:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Cancel Previous Actions
+        uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+      - name: Parse GitHub Ref
+        id: parse_github_ref
+        env:
+          DELIMITER: "/"
+          DEST_BRANCH_FIELD_NUMBER: 2
+          TAG_NAME_FIELD_NUMBER: 3
+          REF: ${{ github.ref }}
+          IGNORE_REF_PREFIX: 'refs/heads/'
+        run: |
+          echo "REF: $REF"
+
+          GITHUB_BRANCH_NAME=${REF#"$IGNORE_REF_PREFIX"}
+          echo "GITHUB_BRANCH_NAME: $GITHUB_BRANCH_NAME"
+
+          DESTINATION_BRANCH=$(echo $GITHUB_BRANCH_NAME | cut -f $DEST_BRANCH_FIELD_NUMBER -d $DELIMITER )
+          TAG_NAME=$(echo $GITHUB_BRANCH_NAME | cut -f $TAG_NAME_FIELD_NUMBER -d $DELIMITER )
+
+          echo "::set-output name=destination_branch::$(echo $DESTINATION_BRANCH )"
+          echo "::set-output name=tag_name::$(echo $TAG_NAME )"
+      - name: Setup Git
+        env:
+          GIT_USER_NAME: soloio-bot
+          GIT_USER_EMAIL: soloio-bot@github.com
+        run: |
+          git config user.name $GIT_USER_NAME
+          git config user.email $GIT_USER_EMAIL
+      - name: Ensure LTS Branch Exists
+        env:
+          BRANCH: ${{ steps.parse_github_ref.outputs.destination_branch }}
+        run: |
+          git fetch origin
+          BRANCH_EXISTS=$(git ls-remote --heads origin $BRANCH)
+
+          if [[ -z ${BRANCH_EXISTS} ]]; then
+            echo "$BRANCH does not exist, creating it"
+            git push origin "HEAD:$BRANCH"
+          else
+            echo "$BRANCH exists, nothing to do"
+          fi
+      - name: Push Commit with Tag Name
```

Comment:
just to clarify: first the github workflow from the `gloo` repo gets called, and pushes the changes to a branch. Then, this workflow here will get called which adds a commit with just a tag name, then opens the PR?

Reply:
Yeah the order of the workflows is:
(1) a gloo release occurs
(2) a gloo GHA updates the APIs and pushes the proposed changes to a branch (ie sync-apis/gloo-branch-that-was-release/gloo-tag-for-release)
(3) a solo-apis GHA notices that a commit is pushed to a `sync-apis` branch and creates a PR 
(4) that PR is manually merged by a developer
(5) a solo-apis GHA notices that a commit is pushed to a LTS branch (ie gloo-v** or gloo-master) and tags the commit

I can update the README to clarify some of these points if you think that would be valuable

Reply:
thanks! yeah i think it would be useful to enumerate the steps as you have above, so that the sequence is clear